go/vt/vttablet/heartbeat/reader.go

```diff
@@ -80,7 +80,6 @@ func (r *Reader) Open(dbc dbconfigs.DBConfigs) {
 	r.mu.Lock()
 	defer r.mu.Unlock()
 	if r.isOpen {
-		log.Fatalf("BUG: Reader object cannot be initialized twice without closing in between: %v", r)
```

Comment:
Here and below: My intuition would be to crash here because a double open or close would be a bug. 

Since it's going to be a non-trivial feature, please document the situations where this can happen :)